.circleci/config.yml

```diff
@@ -99,19 +99,11 @@ jobs:
               ./ci/deploy.sh "$CIRCLE_TAG"
             fi
 
-  docs:
+  docs-build:
     docker:
       - image: circleci/python:3.7
-
-    working_directory: ~/repo
-
     steps:
       - checkout
-
-      # Download and cache python dependencies
-      - restore_cache:
-          key: deps1-{{ .Branch }}-{{ checksum "requirements/docs.txt" }}
```

Comment:
I took this out because it isn't actually a good cache. `docs.txt` includes `default.txt`, which includes `constraints.txt`. That means that we would fail to bust the cache when packages change. I couldn't figure out a clean way to cache the virtualenv and also invalidate based on all the requirements files.

In the future, we could try caching Pip's cache directory instead of the virtualenv. That's safer to under-cache-bust.